src/features/breadcrumbs/BreadcrumbsPanel.tsx

```diff
@@ -28,11 +30,10 @@ export function BreadcrumbsPanel() {
   const clickHandler = items?.length ? breadcrumbItemClick : zoomToTheWorld;
 
   useEffect(() => {
-    const unsubscribe = initSubscriptionToPositionChange();
-    return () => {
-      unsubscribe();
-    };
-  }, []);
+    if (currentMapPosition) {
```

Comment:
Why do you subscribe only when `currentMapPosition `is not `null`?

Reply:
Because debouncedBreadcrumbsUpdate() expects a non-null value in position. It's how old handlePositionUpdate() worked

```
function handlePositionUpdate(position: CurrentMapPositionAtomState) {
  if (position) {
    debouncedBreadcrumbsUpdate(position);
  }
}
```

Reply:
OOO, now I see what got me confused. I thought that subscription inits every time [currentMapPosition] dependency changes. You imported everything from file as `initSubscriptionToPositionChange` and then call `debouncedBreadcrumbsUpdate` from it.
Maybe it's better just to import `debouncedBreadcrumbsUpdate` explicitly from file, you only use this function in Panel?

Reply:
Fixed